README.md

```diff
@@ -6,5 +6,5 @@ The Cockpit Continuous Delivery Tools
  * learn: Machine learning for test results
  * release: container to run releases
  * sink: log sink used by testing infrastructure
- * tests: container used to select and run integration testing
+ * tasks: container used to select and run integration testing
```

Comment:
This does way more these days; how about "run cockpit bots tasks like integration tests or image rebuilds"